src/drivers/ArchiveBuilder.ts

```diff
@@ -56,7 +58,7 @@ function bundleLearningObject(
   const licenseURI = 'https://s3.amazonaws.com/clark-static-assets/Creative+Commons+Legal+Code.pdf';
   return Promise.all([
     bundleURI(licenseURI, archive, 'LICENSE.pdf', prefix),
-    bundleURI(learningObject.materials['pdf'].url, archive, learningObject.materials['pdf'].name, prefix),
+    bundleURI(learningObject.materials.pdf.url ? learningObject.materials.pdf.url : generatePDFURL(learningObject.id), archive, learningObject.materials.pdf.name, prefix),
```

Comment:
This can be shortened to
```suggestion
    bundleURI(learningObject.materials.pdf.url || generatePDFURL(learningObject.id), archive, learningObject.materials.pdf.name, prefix),
```